cmd/containerd/config_solaris.go

```diff
@@ -0,0 +1,21 @@
+// +build solaris
+
+package main
+
+import (
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/server"
+)
+
+func defaultConfig() *server.Config {
+	return &server.Config{
+		Root: "/var/lib/containerd",
+		GRPC: server.GRPCConfig{
+			Address: containerd.DefaultAddress,
+		},
+		Debug: server.Debug{
+			Level:   "info",
+			Address: "/var/run/containerd/debug.sock",
```

Comment:
sure.  thanks for pointing this out, because it also made me realize that i should be getting DefaultAddress from the server package instead of the containerd package.  so i'll fix that as well.